src/www/style.css

```diff
@@ -14,6 +14,10 @@
  * limitations under the License.
  */
 
+ body {
```

Comment:
we're not properly CSS resetting, which why the default browser styling is causing that weird margin inset

Reply:
Maybe add a comment?

Reply:
Shall we include other CSS resetting properties in the future (e.g., https://github.com/necolas/normalize.css/blob/master/normalize.css)?

Reply:
Normalize is not a reset, but yes, I'm planning on that. We need postcss.